### PR TITLE
Add missing changelog entry

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.6.3 (2021-06-08)
+
+- Fixed to back to top component to hide the component if the `buttonHidden` option is enabled after the component is rendered. [#219](https://github.com/blackbaud/skyux-layout/pull/219)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]: Is `component` correct here? Looking at the docs at https://developer.blackbaud.com/skyux-layout/docs/back-to-top?docs-active-tab=development, I assume this should be `directive`.

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Good catch. There is technically a component but the consumer uses the directive to show the component and the actual change was in the directive. I've fixed it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 4.6.3 (2021-06-08)
+
+- Fixed to back to top directive to hide the component if the `buttonHidden` option is enabled after the component is rendered. [#219](https://github.com/blackbaud/skyux-layout/pull/219)
```
Comment (blackbaud-paulah) [inline CHANGELOG.md:3]:
```suggestion
- Fixed the back to top directive to hide the component if the `buttonHidden` option is enabled after the component is rendered. [#219](https://github.com/blackbaud/skyux-layout/pull/219)
```

Comment (blackbaud-johnly) [reply CHANGELOG.md:3]: Nice catch! I totally overlooked that.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.6.3](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.6.3)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.6.3](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.6.3)

**Review by blackbaud-paulah — DISMISSED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.6.3](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.6.3)

**Review by blackbaud-johnly — APPROVED** [review]

**Review by blackbaud-paulah — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.6.3](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.6.3)

**Pull request merged**